lib/src/core/models/chat/sub_models/choices/sub_models/message.dart

```diff
@@ -28,7 +33,10 @@ final class OpenAIChatCompletionChoiceMessageModel {
     return OpenAIChatCompletionChoiceMessageModel(
       role: OpenAIChatMessageRole.values
           .firstWhere((role) => role.name == json['role']),
-      content: json['content'],
+      content: json['content'] ?? '',
```

Comment:
Nice move with this.

Reply:
Yeah, don't like it, would have preferred to do content as String? but that'd be a breaking change. I think for now this is fine.

Reply:
yeah, it will be a breaking change if it will be a `String?`, we don't want also to set an opinionated sentence so as you said, it would be fine like this currently, thank you.